crawl-ref/source/dat/des/branches/crypt.des

```diff
@@ -1463,6 +1486,884 @@ cG.MGM.GcG.1.GcG.1.GcG.1.Gc
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+Q-...........................-Q
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+x.....x.x.....x.x.....x.x.....x
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+Q-...........-x.x-...........-Q
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x
+xxxxxxxQxxxxxxxQxxxxxxxQxxxxxxx
+ENDMAP
+
+NAME:   nicolae_crypt_golgotha
+TAGS:   transparent
+DEPTH:  Crypt
+KMONS:  S = curse skull
+KMONS:  s = flying skull
+SUBST:  S = SSsss, s = sss.
+MAP
+     .........
+     ..x.x.x..
+     .x.x.x.x.
+     .........
+     .s.xxx.s.
+.........x.........
+..x.s.xx...xx.s.x..
+.x....xx.S.xx....x.
+..x.x.........x.x..
+.x..xx.S.S.S.xx..x.
+..x.x.........x.x..
+.x....xx.S.xx....x.
+..x.s.xx...xx.s.x..
+.........x.........
+     .s.xxx.s.
+     .........
+     .x.x.x.x.
+     ..x.x.x..
+     .........
+ENDMAP
+
+NAME:  nicolae_crypt_grand_chamber
+TAGS:  transparent
+DEPTH: Crypt
+KMONS: E = guardian mummy / death scarab w:5 / ushabti w:5 / nothing w:30
+KMONS: P = mummy priest
+SUBST: * = ***|
+MAP
+xxxxxxxxxxxxxxxxxxxxxx
+xxxx*x.x.x.xxxxxxxxxxx
+xxx*..E.E.E.xxxxxxxxxx
+xx*x.x.x.x.x.x.xxxxxxx
+x*..E.E.E.E.E.E.xxxxxx
+xx.x.....x.x.x.x.x.x.x
+x*..E.P.E.E.E.E......+
+xx.x.....x.x.x.x.x.x.x
+x*..E.E.E.E.E.E.xxxxxx
+xx*x.x.x.x.x.x.xxxxxxx
+xxx*..E.E.E.xxxxxxxxxx
+xxxx*x.x.x.xxxxxxxxxxx
+xxxxxxxxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:    nicolae_crypt_grave_offerings
+TAGS:    transparent
+DEPTH:   Crypt
+KITEM:   a = $ / % / * / | w:5
+KMONS:   a = 0 / 9 w:5 / nothing w:30
+SHUFFLE: DEFH, JK
+SUBST:   D : @, E : @x, F : @xx, H : @xxx, J : x.., K : ., L : x..
+MAP
+   xxxEEExxx
+  xx.......xx
+ xx.........xx
+xx...JK.KJ...xx
+x...LxxGxxL...x
+D....xaaa+....F
+D....xaaa+....F
+x...LxxGxxL...x
+xx...JK.KJ...xx
+ xx.........xx
+  xx.......xx
+   xxxHHHxxx
+ENDMAP
+
+NAME:   nicolae_crypt_heckumenical_temple
+TAGS:   transparent
+DEPTH:  Crypt
+KMONS:  9 = 9
+KFEAT:  9 = Y
+KFEAT:  O = altar_beogh
+KFEAT:  D = altar_dithmenos
+KFEAT:  K = altar_kikubaaqudgha
+KFEAT:  L = altar_lugonu
+KFEAT:  M = altar_makhleb
+KFEAT:  y = altar_yredelemnul
+NSUBST: _ = O / D / K / L / M / y
+MAP
+xxxxxxxxxxxxxxx
+xxxx._.x._.xxxx
+xxxx...x...xxxx
+xxxxx+xxx+xxxxx
+x..x..xxx..x..x
+x_.+.0.x.0.+..+
+x..xx.....xx..x
+xxxxxx.9.xxxxxx
+x..xx.....xx..x
+x_.+.0.x.0.+..+
+x..x..xxx..x..x
+xxxxx+xxx+xxxxx
+xxxx...x...xxxx
+xxxx._.x._.xxxx
+xxxxxxxxxxxxxxx
+ENDMAP
+
+# The lich isn't *that* imprisoned. I mean, there's stairs.
+NAME:   nicolae_crypt_imprisoned_lich
+TAGS:   no_descent
+DEPTH:  Crypt, !Crypt:$
+ORIENT: west
+KITEM:  r = % w:40 / *
+KITEM:  R = * w:40 / |
+KITEM:  V = |
+KMONS:  r = death knight / skeletal warrior / vampire mage / deep elf death mage
+KMONS:  R = revenant / curse skull / vampire knight / ancient champion
+KMONS:  V = lich
+KFEAT:  V = V
+KFEAT:  - = iron_grate
```

Comment:
I know glass isn't as evocative of a prison as iron grates are, but it's very silly and slow to abuse the grate to LRD / Airstrike / Freezing Cloud / Ignition enemies that don't know how to get past it aside from vampire mage summons, so I'd rather use transparent rock instead.